Resources/About Contents.html

```diff
@@ -54,66 +59,69 @@ <h5>VLCFORIOSVERSION<br /><br />MOBILEVLCKITVERSION</h5>
-Copyright &copy; 2008-2013 Plausible Labs Cooperative, Inc., &copy; 2010 MOSO Corporation, Pty Ltd., &copy; 2012 HockeyApp, Bit Stadium GmbH. - <a href="#mit">MIT License</a><br />
+Copyright &copy; 2012-2015 Denis Hennessy (Peer Assembly) - <a href="#bsd3clause">3-clause BSD License</a><br />
 <br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/adobe/source-code-pro">Raleway</a><br />
 Copyright &copy; 2012-2013 The League of Moveable Type, Matt McInerney, Pablo Impallari, Rodrigo Fuenzalida, Igino Marini. - <a href="#silopenfont">SIL Open Font License, Version 1.1</a><br />
 <br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://developer.apple.com/Library/ios/samplecode/Reachability/Introduction/Intro.html">Reachability</a><br />
-Copyright &copy; 2013 Apple Inc. All Rights Reserved. - <a href="#applesampledisclaimer">Apple Sample Code disclaimer</a><br />
+Copyright &copy; 2013-2016 Apple Inc. All Rights Reserved. - <a href="#applesampledisclaimer">Apple Sample Code disclaimer</a><br />
 <br />
-&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/karlvr/XKKeychain">XKKeychain</a><br />
-Copyright &copy; 2014-2016 Karl von Randow - <a href="#mit">MIT License</a><br />
-<br />
-&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/xmartlabs/XLPagerTabStrip">XLPagerTabStrip</a><br />
-Copyright &copy; 2014-2018 xmartlabs - <a href="#mit">MIT License</a><br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/realm/SwiftLint">SwiftLint</a><br />
+Copyright &copy; 2015-2019 Realm Inc - <a href="#mit">MIT License</a><br />
 <br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/ndfred/xcode-tools/blob/master/update_strings.py">update_strings.py</a><br />
 Copyright &copy; 2010 Frédéric Sagnes - <a href="#mit">MIT License</a><br />
 <br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/fkuehne/upnpx">upnpx</a><br />
-Copyright &copy; 2011-2013 Bruno Keymolen, Jonathan Guan, Mujtaba Hassanpur, &copy; 2013-2017 Felix Paul K&uuml;hne - <a href="#bsd3clause">3-clause BSD License</a><br />
+Copyright &copy; 2011-2013 Bruno Keymolen, Jonathan Guan, Mujtaba Hassanpur, &copy; 2013-2019 Felix Paul K&uuml;hne - <a href="#bsd3clause">3-clause BSD License</a><br />
 <br />
 &nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/fkuehne/WhiteRaccoon">WhiteRaccoon</a><br />
 Copyright &copy; 2011 Valentin Radu, &copy; 2013 Felix Paul K&uuml;hne - <a href="#mit">MIT License</a><br />
 <br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/karlvr/XKKeychain">XKKeychain</a><br />
+Copyright &copy; 2014-2016 Karl von Randow - <a href="#mit">MIT License</a><br />
+<br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://github.com/xmartlabs/XLPagerTabStrip">XLPagerTabStrip</a><br />
+Copyright &copy; 2014-2019 xmartlabs - <a href="#mit">MIT License</a><br />
+<br />
+&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://aomedia.googlesource.com/aom">aom</a><br />
+Copyright &copy; 2016-2019, Alliance for Open Media - <a href="#bsd2clause">BSD-2</a><br />
+<br />
```

Comment:
Ouch. We should ship dav1d exclusively, but that's a separate issue.

Reply:
yeah we can do that in VLCKit and update the about screen, I'll file a ticket on VLCKit